Tests/Smoketest3.xctestplan

```diff
@@ -34,16 +34,6 @@
         "FindInPageTests",
         "FirstRunTourTests",
         "FxScreenGraphTests",
-        "HistoryTests\/testAllOptionsArePresent()",
-        "HistoryTests\/testClearHistoryFromSettings()",
-        "HistoryTests\/testClearRecentlyClosedHistory()",
-        "HistoryTests\/testEmptyHistoryListFirstTime()",
-        "HistoryTests\/testLongTapOptionsRecentlyClosedItem()",
-        "HistoryTests\/testOpenInNewPrivateTabRecentlyClosedItem()",
-        "HistoryTests\/testOpenInNewTabRecentlyClosedItem()",
-        "HistoryTests\/testOpenSyncDevices()",
-        "HistoryTests\/testPrivateClosedSiteDoesNotAppearOnRecentlyClosed()",
-        "HistoryTests\/testRecentlyClosedOptionAvailable()",
```

Comment:
And same here, not all History tests have to run in all tests plan, we have the Full_Functional and the Smoketest, I would sugges to not change this for now so that we have the tests running for the Smoke or Full Functional as they were, that has not changed.

Reply:
Some of the tests' names have updated. I will note the new names.

Reply:
Yeah, thank you. If you have broken down a test that was part of the smoketest in smaller tests, all of them should be added to the smoke test, and similarly for a test that was part of the full functional test suite